sdk/eventhub/Microsoft.Azure.EventHubs.Processor/src/Microsoft.Azure.EventHubs.Processor.csproj

```diff
@@ -21,6 +21,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.KeyVault.Core" />
     <PackageReference Include="Microsoft.Azure.Storage.Blob" />
-    <PackageReference Include="Newtonsoft.Json" />
+    <PackageReference Include="NetStandard.Library" VersionOverride="2.0.3"/>
+    <PackageReference Include="Newtonsoft.Json" VersionOverride="13.0.1"/>
```

Comment:
What's our test plan for this, and why make it local rather than at the root level?